play/jaffle-shop-dbt/README.md

```diff
@@ -0,0 +1,149 @@
+[Materialize]: https://materialize.com/
+[`dbt-materialize`]: https://pypi.org/project/dbt-materialize/
+[`wikirecent-dbt` demo]: https://github.com/MaterializeInc/materialize/blob/main/play/wikirecent-dbt/README.md
+[our docs]: https://materialize.com/docs/
+
+## Setting up a `jaffle_shop` with Materialize
+
+Setting up the `jaffle_shop` project with Materialize is similar to setting it up with any other
+data warehouse. The following instructions are based off the [traditional `jaffle_shop`] steps with a few
+Materialize-specific modifications:
+
+1. Following [the usual `jaffle_shop` instructions], install dbt, clone the `jaffle_shop` repository, and navigate
+   to the cloned repo on your machine.
+
+1. In your cloned `dbt_project.yml`, make the following changes to the [model materializations]:
+    ```nofmt
+    models:
+        jaffle_shop:
+            marts:
+                core:
+                    materialized: materializedview
+                    intermediate:
+                        materialized: view
+            staging:
+                materialized: view
+                tags: ["staging", "hourly"]
+    ```
+    Tip: Only materializing your `core` business models as materialized views, without materializing
+    your intermediate or staging views, ensures that you're only using the memory you need in
+    Materialize.
+
+1. Install the [dbt-materialize plugin]. You may wish to do this within a Python virtual environment on your machine:
+    ```bash
+    python3 -m venv dbt-venv
+    source dbt-venv/bin/activate
+    pip install dbt-materialize
+    ```
+
+1. [Install and run Materialize].
+
+1. Create a `jaffle_shop` [dbt profile] that will connect to Materialize. The following profile will
+   connect to a Materialize instance running locally on your machine. The `host` parameter will need
+   to be updated if it's self-hosted in the cloud or run with Docker:
+    ```nofmt
+    jaffle_shop:
+        outputs:
+            dev:
+            type: materialize
+            threads: 1
+            host: localhost
+            port: 6875
+            user: materialize
+            pass: password
+            dbname: materialize
+            schema: jaffle_shop
+
+        target: dev
+    ```
+    > If the `profiles.yml` you're using for this project is not located at `~/.dbt/`, you will have to provide [additional information] to use the `dbt` commands later on.
+
+1. Check that your newly created `jaffle_shop` profile can connect to your Materialize instance:
+   ```bash
+   dbt debug
+   ```
+
+1. Load the static `jaffle_shop` data into Materialize:
+    ```bash
+    dbt seed
+    ```
+
+1. Run the provided models:
+    ```bash
+    dbt run
+    ```
+
+1. In a new shell, connect to Materialize to check out the `jaffle_shop` data you just loaded:
+    ```bash
+    # Connect to Materialize
+    psql -U materialize -h localhost -p 6875
+
+    # See all the newly created views
+    materialize=> SHOW VIEWS IN jaffle_shop;
```

Comment:
I don't spend enough time in Markdown but is there a way to show the command and then the output separately? Just so the user can more easily differentiate what is happening.

Reply:
I totally see what you mean! I'm also not a Markdown expert. I played with a few things, and just ended up putting in extra space. I think it helps? I'll ask our docs team for their thoughts as well.

Reply:
Yes, I do think that helps. Perhaps if we can't have separate boxes, just commenting out `##output` or something like that